packages/seller-client/src/components/Price/Price.tsx

```diff
@@ -4,7 +4,7 @@ type PriceProps = {
   price: number;
 };
 const Price: React.FC<PriceProps> = ({ price }) => {
-  return <span>${price.toFixed(2)}</span>;
+  return <span>{`${price < 0 ? '-' : ''}$${Math.abs(price.toFixed(2))}`}</span>;
```

Comment:
Looks like you're calling `Math.abs` on the string returned by `price.toFixed(2)` here!

```suggestion
  return <span>{`${price < 0 ? '-' : ''}$${Math.abs(price).toFixed(2)}`}</span>;
```

Reply:
They actually both have the same output (I tested to make sure). But your way looks better so I committed it!